### PR TITLE
Fixes Prisoner accounts and some refactors

### PR DESCRIPTION
**Pull request by Bilbo367**
## About The Pull Request

Refactors some code. and deletes ACCOUNT_PRISON so prisoners can have money.

## Why It's Good For The Game

Lets prisoners have money.

### CODE REVIEW

File: fulp_modules/_defines/fulp_defines.dm [hunk 1]
```diff
@@ -51,6 +41,3 @@
 
 ///Define for the 'Rabbits' Faction.
 #define FACTION_RABBITS "rabbits"
-
-//Define for the Syndicate Engineer Ruin, used in 'syndicate_engineer.dm"
-#define ROLE_SYNDICATE_ENGINEER "Syndicate Engineer"
```
Comment (JohnFulpWillard) [inline fulp_modules/_defines/fulp_defines.dm:56]: What’s the reason for removing the define?

Comment (Bilbo367) [reply fulp_modules/_defines/fulp_defines.dm:56]: We could keep it, its only used in one place so it seems weird to define it for only one use.

**Comment by remove32** [comment]
filthy prisoners

File: fulp_modules/_defines/fulp_defines.dm (hunk 1 above)
Comment (JohnFulpWillard) [reply fulp_modules/_defines/fulp_defines.dm:56]: I think you should keep it in

**Comment by Bilbo367** [comment]
let me fix my master

**Pull request closed without merging**